internal/faustjs.org/docs/guides/pagination.mdx

```diff
@@ -0,0 +1,99 @@
+
+## Setup
+
+To get started, create a folder in the root of your Faust.js project called `plugins`.  In this `plugins` folder, create a file called `RelayStylePagination.js` and insert this content:
+
+```
+import { relayStylePagination } from "@apollo/client/utilities";
+
+export class RelayStylePagination {
+  apply(hooks) {
+    const { addFilter } = hooks;
+
+    addFilter("apolloClientInMemoryCacheOptions", "faust", (options) => {
+      return {
+        ...options,
+        typePolicies: {
+          ...options.typePolicies,
+          RootQuery: {
+            ...options.typePolicies.RootQuery,
+            fields: {
+              posts: relayStylePagination(),
+            },
+          },
+          ContentType: {
+            fields: {
+              contentNodes: relayStylePagination(),
+            },
+          },
+        },
+      };
+    });
+  }
+}
+```
+The Apollo client can implement [relay-style pagination](https://www.apollographql.com/docs/react/pagination/cursor-based/) with the relay spec using merge and read functions, which means all the details of `connections`, `edges` and `pageInfo` can be abstracted away, into a single, reusable helper function. WPGraphQL follows the relay spec as well.
+We are importing the `relayStylePagination` function from Apollo and then a `class component` is created which is the basic syntax of the Faust plugin.
+
+Next, we have an `apply` method with the `hooks` parameter which is an object that gives you a function called `addFilter`. The `addFilter` function allows us to modify the [Apollo Client’s `InMemoryCacheOptions`](https://www.apollographql.com/docs/react/caching/cache-configuration/).
+
+The `addFilter` hook is taken and calls the memory cache function options. The options are coming from the Apollo Client Cache configuration. These options allow for configuring the cache for  pagination. The faust namespace follows that.
+
+## Register the Plugin for Pagination
+
+Import and register your plugin for pagination in `faust.config.js`
+
+```
+import { setConfig } from "@faustwp/core";
+import templates from "./wp-templates";
+import possibleTypes from "./possibleTypes.json";
+import { RelayStylePagination } from "./plugins/RelayStylePagination";
+
+/**
+ * @type {import('@faustwp/core').FaustConfig}
+ **/
+export default setConfig({
+  templates,
+  experimentalPlugins: [new RelayStylePagination()],
+  possibleTypes,
+});
+```
+
+## Example WPGraphQL Pagination Query
+
+```
+const GET_POSTS = gql`
+  query getPosts($first: Int!, $after: String) {
+    posts(first: $first, after: $after) {
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+      edges {
+        node {
+          id
+          databaseId
+          title
+          slug
+        }
+      }
+    }
+  }
+`;
+```
+This is an example of a query with the fields for pagination being `hasNextPage` and `endCursor`.
+
+
```

Comment:
I think we should include how to query for this data too. For instance, how they can get the `fetchMore` function like in the blueprint:

https://github.com/wpengine/atlas-blueprint-portfolio/blob/f6d4d5525711e307392b0f25501806b7486270d7/pages/posts/index.js#L22-L24

Reply:
Good call out on this, would you want me to specifically use the Blue Print example or can I use my raw Faust.js example?